salt/pillar/profiles.sls

```diff
@@ -22,6 +22,10 @@ profiles:
         level: DEBUG
     orcid_dummy:
         pinned_revision_file: /srv/profiles/orcid-dummy.sha1
+    sns:
+        name: profiles
```

Comment:
I meant the other way around: your name was the correct one (compare https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L994) and the `aws sns` command that creates a topic for testingneeds to use this pillar value